web/src/views/ConsentPortal/OpenIDConnect/DecisionFormScopes.tsx

```
@@ -0,0 +1,83 @@
+
+const DecisionFormScopes: React.FC<Props> = (props: Props) => {
+    const { t: translate } = useTranslation(["consent"]);
+
+    const { classes } = useStyles();
+
+    return (
+        <Grid size={{ xs: 12 }}>
+            <Box className={classes.scopesListContainer}>
+                <List className={classes.scopesList}>
+                    {props.scopes.map((scope: string) => (
+                        <Tooltip title={translate("Scope", { name: scope })}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)